README.md

```diff
@@ -1,8 +1,9 @@
-# Unofficial Patch 9.1f for Metro for Steam skin
+# Unofficial Patch 9.1g for Metro for Steam skin
 
-This unofficial patch for [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for unhindered and customized design.
 
-Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Unofficial Patch thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
+This unofficial patch for the [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for customizable productivity and aesthetics.
+
+Additional details and feedback are available on the [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```

Comment:
```suggestion
Additional details and feedback are available on the [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```
let's keep "community".

Reply:
btw, lose the extra spacing below the headline too, introduced by ur commits.

Reply:
I'd like to hear @redsigma's opinion on it

Reply:
haven't he already decided, by making a commit introducing this into the file directly? :)
it was ur original proposal... no?

Reply:
I proposed to remove it in 8839a94, my original commit.

Reply:
hmm, interesting... I actually thought it was missing originally.
anyway, I would guess he already decided since he didn't introduce it himself, but otherwise...
I'm not sure why it's so important to remove it.

in any case, I'm ok with the other changes.